cypress-io/cypress
# chore: adding runnables to protocol

### Additional details


This PR adds the `runnables` to the protocol.

### Steps to test


- Clone the `cypress-services` repo and checkout `mschile/protocol/add_runnables`
  - Run `yarn`
  - Run `yarn watch` in `packages/app-capture-protocol`
- Set `CYPRESS_LOCAL_PROTOCOL_PATH` to the path to `cypress-services/packages/app-capture-protocol/dist/index.js`
- Clone the `cypress` repo  and checkout `mschile/protocol/add_runnables`
  - Run `yarn`
  - Execute `cypress run` on a project in record mode with `DEBUG` set to `cypress:*protocol*` and ensure the `runnables` are added to the database.

### How has the user experience changed?

<img width="691" alt="Screenshot 2023-03-21 at 8 56 21 PM" src="https://user-images.githubusercontent.com/2002044/226790475-fda36b32-0e91-4f96-a806-557832911707.png">

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
